arduino/opencr_arduino/opencr/libraries/DynamixelWorkbench/src/dynamixel_workbench_toolbox/dynamixel_driver.cpp

```diff
@@ -863,6 +883,25 @@ bool DynamixelDriver::readRegister(uint8_t id, const char *item_name, int32_t *d
   }
   else
   {
+    switch (control_item->data_length)
```

Comment:
@t-kitajima  Is this switch-case statement can erase after change?

Reply:
@routiful Yes. You can delete that switch-case.

Reply:
I have changed the code you suggest and the functions are working well.
Thank you for your contribution. 

https://github.com/ROBOTIS-GIT/OpenCR/commit/4dfdf1ae642e8dd9944d4eeae1243e831807311a